diff -Naur c_icap-0.5.10.old/configure.ac c_icap-0.5.10/configure.ac
--- c_icap-0.5.10.old/configure.ac	2022-09-15 14:25:31.815611889 +0200
+++ c_icap-0.5.10/configure.ac	2022-09-15 15:49:16.183587057 +0200
@@ -322,7 +322,7 @@
 
 if test a"$perlbin" != a; then
 	perlcore=`$perlbin -MConfig -e 'print $Config{archlib}'`/CORE;
-        perllib=`$perlbin -MConfig -e 'print $Config{libs}'`;
+        perllib=`$perlbin -MConfig -e 'print $Config{libs}' | sed -e 's/-ldb//g'`;
         perlccflags=`$perlbin -MConfig -e 'print $Config{ccflags}'`;
 	perlldflags=`$perlbin -MConfig -e 'print $Config{ccdlflags}'`;
 fi
@@ -524,7 +524,7 @@
 if test "a$libdb" != "ano"; then
    if test "a$libdbpath" != "a"; then
      CFLAGS="-I$libdbpath/include $CFLAGS"
-     LDFLAGS="-L$libdbpath/lib $LDFLAGS"
+     LDFLAGS="-L$libdbpath/lib64 $LDFLAGS"
    fi
 
    # We are going to see if we can found a Berkeley DB located under a
